cogs/help.py

```diff
@@ -31,7 +31,7 @@
 HELP_MOD_ID: int = 896860382226956329
 GUILD_ID: int = 881118111967883295
 CUSTOM_ID_PREFIX: str = "help:"
-NAME_TOPIC_REGEX: str = r"(^.*?) \((.*?[0-9]{4})\)$"
+NAME_TOPIC_REGEX: str = r"^(.*?)(\((?:[^)])*\)[^(]*$)"
```

Comment:
Fix to not include the parentheses and space in the author.

This also uses named groups, so later, when groups are accessed, it should be .group("topic") / .group("author") instead of .group(1) / .group(2)
```suggestion
NAME_TOPIC_REGEX: str = r"^(?P<topic>.*?) \((?P<author>[^)]*[^(]*)\)$"
```

Reply:
Thanks!